server/database/config/build.sql

```diff
@@ -0,0 +1,74 @@
+BEGIN;
+
+DROP TABLE IF EXISTS users,parent,teacher,student,teacher-student,subjects,subjects-teacher,
```

Comment:
use a singular name for database tables

```suggestion
DROP TABLE IF EXISTS 'user',parent,teacher,student,teacher-student,subjects,subjects-teacher,
```